src/components/select/select.ts

```diff
@@ -365,6 +364,17 @@ export class Select extends BaseInput<string[]> implements AfterViewInit, OnDest
     });
   }
 
+  /**
+   * Close the select interface.
+   */
+  close() {
+    if (!(this._overlay && this.isFocus())) {
+      return;
+    }
+
+    this._overlay.dismiss();
```

Comment:
I think you should return the promise returned here. (otherwise some async stuff can't be handled).